tox.ini

```diff
@@ -54,3 +55,24 @@ commands: {[common-lint]commands}
 [testenv:py{37,38,39,310,311}-lint]
 extras: {[common-lint]extras}
 commands: {[common-lint]commands}
+
+[testenv:py{37,38,39,310,311}-wheel]
+basepython=
+    py37: python3.7
```

Comment:
This remapping of the base python shorthand to the version isn't necessary since it's defined in the `testenv` section above.

Reply:
space added